docs/compatibility.md

```diff
@@ -1,263 +1,187 @@
+| - source                     |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - target                     |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - uid                        |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - gid                        |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - mode                       |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| services.<name>              |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - build                      |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - args                     |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - cache_from               |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - context                  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - dockerfile               |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - extra_hosts              |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - labels                   |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - network                  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - shm_size                 |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+|   - target                   |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cap_add                    |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cap_drop                   |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  C  |  C  |  C  |  C  |  C  |  C  |
+| - cpu_shares                 |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - cpu_quota                  |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - cpuset                     |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - command                    |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - configs                    |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
+| - cgroup_parent              |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - container_name             |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - credential_spec            |  *  |  *  |  *  |  *  |  *  |     |     |     |     |     |     |     |     |
```

Comment:
Note that support for `credential_spec` in Compose is pending: https://github.com/docker/compose/issues/5488